Gemfile

```diff
@@ -3,7 +3,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.2.2"
 
-gem "rails", "~> 7.0.4.3"
+gem "rails", "~> 7.0", ">= 7.0.4.3"
```

Comment:
@joemasilotti : it may be that dependabot wasn't finding the new version because of the way you had the version here.
The Rubygems recommendation is to have both `~>` and a `>=` (ref: https://rubygems.org/gems/rails/versions/7.0.4.3).

This is why, in my first push on this PR, I've had 2 commits one to just change the rails version here.

One side effect of this current value, is that it will update automatically from 7.0.4 to 7.0.5.
With this version, when I run `bundle update rails` today, it updated the version from 7.0.5.1 to 7.0.6 (release 2 weeks days ago).